x/oracle/simulation/operations.go

```diff
@@ -496,3 +510,145 @@ func SimulateMsgDeleteTask(ak types.AccountKeeper, bk types.BankKeeper, contract
 		return simtypes.NewOperationMsg(msg, true, "", nil), nil, nil
 	}
 }
+
+// SimulateMsgCreateTxTask generates a MsgCreateTxTask object with all of its fields randomized.
+// This operation leads a series of future operations.
+func SimulateMsgCreateTxTask(ak types.AccountKeeper, k keeper.Keeper, bk types.BankKeeper) simtypes.Operation {
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simtypes.Account, chainID string) (
+		simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		creator, _ := simtypes.RandomAcc(r, accs)
+		creatorAcc := ak.GetAccount(ctx, creator.Address)
+		bounty := simtypes.RandSubsetCoins(r, bk.SpendableCoins(ctx, creatorAcc.GetAddress()))
+		randomDuration := time.Duration(rand.Int63n(int64(2000 * time.Minute)))
+		validTime := ctx.BlockTime().Add(randomDuration)
```

Comment:
Do we need to add some operations after tx-task finish? Tx-task may stay in pending with this time duration.

Reply:
At present, it should contain all operations, `delete task `and `claim reward`